.github/workflows/gosec.yml

```diff
@@ -6,7 +6,8 @@ on: # yamllint disable-line rule:truthy
   workflow_call:
 
 permissions:
-  pull-requests: read
+  security-events: write
```

Comment:
ref https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/controlling-permissions-for-github_token which is interesting, as a PR level permission impacts security events. but meanwhile https://github.com/sustainable-computing-io/kepler/actions/runs/12065944361 daily job works, I suppose that at the merge event, the PR info is used. Anyway, in this PR updated both gosec and daily job with same permission.